### DISCUSSION TITLE
Dark Mode?

### DISCUSSION THREAD

**Discussion by Some1OnLine** [drakkan/sftpgo#804] [Ideas]
I just came across this nice little gem. This is basically exactly what I have been looking for. I see I can maybe change colors etc.. via css file but is there by any chance a dark mode/toggle/setting already baked in for web client or admin panel? If not any chances of adding one for those of us with a hard time with lots of bright colors?

Thank you for the hard and open work!

~SOL

**Comment by drakkan** [comment]
Hi,

SFTPGo uses  [SB Admin 2](https://github.com/StartBootstrap/startbootstrap-sb-admin-2) and this theme does not have a dark mode, sorry. A dark mode, or an easier way to customize colors, is something I want to add in the future. Let's see if SB Admin 2 will be improved  or if I will have to switch to a different theme, for example [AdminLTE3](https://adminlte.io/themes/v3/)

**Reply by 10kc-svats** [reply]
@drakkan so for now SFTPGo does not support changing colors and the page title?

**Reply by firestrife23** [reply]
I like AdminLTE3, seem to have more options tho. I even tried changing SB Admin 2 CSS file, and I'm not getting the result I want. Because there are too many stuffs in there to change.

**Comment by Some1OnLine** [comment]
All good I understand. Thanks for responding! I will be watching and following this project going forward.

~SOL

**Comment by 10kc-svats** [comment]
Hey @drakkan, yeah, it would be cool to have simple CSS customization options, like:
- Adding a company logo
- Updating the page title
- Changing colors & fonts

**Reply by drakkan** [reply]
> Hey @drakkan, yeah, it would be cool to have simple CSS customization options, like:
> 
> * Adding a company logo
> * Updating the page title
> * Changing colors & fonts

This is supported, please read the docs before commenting on an old thread and check the [support policy](https://github.com/drakkan/sftpgo#support-policy)
